src/libostree/libostree.sym

```diff
@@ -352,8 +352,8 @@ global:
  */
 
 /* UNCOMMENT WHEN ADDING THE FIRST NEW SYMBOL FOR 2016.8
-LIBOSTREE_2016.7 {
+LIBOSTREE_2016.8 {
```

Comment:
I'm pretty sure I psyched myself out on this originally - I stared at it for 10 seconds and convinced myself it was wrong, then changed it when I shouldn't have.  Tip for future me: Don't overthink it.